### DISCUSSION TITLE
PR, Commit, issue 컨벤션은 통일해야 편할 것 같습니다.

### DISCUSSION THREAD

**Discussion by sokojh** [heading1/WYLSBingsu#6] [Ideas]
그동안 PR, commit, issue 등 각자 스타일이 다양하다보니

다시 볼때 가독성이 너무 떨어져서 생산성 하락을 가져오는 것 같습니다.

이번에 컨벤션을 통일해서 생산성을 높이면 좋을 것 같습니다.

예시
|적용 전 |적용 후|
|------|---|
|<img width="417" alt="스크린샷 2022-06-06 오후 8 28 35" src="https://user-images.githubusercontent.com/83265848/172152514-0b84dbc1-14c8-45d7-a094-19ceca8111b3.png">|<img width="718" alt="스크린샷 2022-06-06 오후 8 29 02" src="https://user-images.githubusercontent.com/83265848/172152526-6550c850-ab3a-4059-b284-bce996518755.png">

**Comment by Circlewee** [comment]
다른 레포지토리를 참고해서 템플릿을 만들어 보겠습니다!!

**Reply by sokojh** [reply]
> 

좋습니다.
템플릿이 있어야 된다고 생각합니다!   🤗
